setup.py

```diff
@@ -34,6 +34,12 @@
 with open(os.path.join(here, "README.rst")) as f:
     long_description = f.read()
 
+if sys.platform == 'win32':
+    kwds = {'data_files': [("cmd", ["cmd/activate", "cmd/deactivate",
+                                    "cmd/activate.bat", "cmd/deactivate.bat"]), ]
+            }
+else:
+    kwds = {'data_files':  [("cmd", ["cmd/activate", "cmd/deactivate"]), ]}
```

Comment:
This is done specifically to mimic the behavior of git for windows.  See https://github.com/conda/conda/pull/1727#discussion_r42775466

Please suggest something else if you'd like to break that convention.